omegaconf/_utils.py

```diff
@@ -349,9 +349,8 @@ def _is_missing_value(value: Any) -> bool:
 
 
 def _is_missing_literal(value: Any) -> bool:
-    from omegaconf import MISSING, Node
+    from omegaconf import MISSING
```

Comment:
Here's an alternate version that squeezes even more juice out of it:

```python
MISSING = None
def _is_missing_literal(value: Any) -> bool:
    global MISSING
    if MISSING is None:
        from omegaconf import MISSING

    return isinstance(value, str) and value == MISSING
```

Reply:
Actually ~twice as fast:
```python
MISSING = "???"
def _is_missing_literal(value: Any) -> bool:
    return isinstance(value, str) and value == MISSING
```

=> I suggest to define MISSING in a different file that can be imported without circular dependencies

Reply:
nice.
Or actually switch back to using literal "???" in hotspots.
MISSING was defined to be used with Structured Configs.

Reply:
Using literal is 8x faster for this function (compares to master):

![image](https://user-images.githubusercontent.com/376455/111719736-b8262680-8819-11eb-8b25-8835a643e1fc.png)

Reply:
Yeah `"???"` is the shortcut solution -- FYI it's the same speed as defining `MISSING = "???"` so if we can import it without circular dependencies it should be equivalent (and it's a bit cleaner). But either works.

Reply:
See followup diff.